### ISSUE TITLE
Responsive mode doesn't work in smartphones help please!!!

### ISSUE THREAD

**Issue by Dreadchild** [ignacionelson/ProjectSend#7]
Responsive mode doesn't work in smartphones help please!!!

How can i Fix that problem?

**Comment by lenamtl** [comment]
+1

**Comment by lenamtl** [comment]
Suggestion:
 
We should update to Bootstrap v 3+ first

Then use :

Configuration Tabs cn be changed for Bootstrap-Tabdrop 
http://www.eyecon.ro/bootstrap-tabdrop/

Pupload: https://github.com/leedorian/pupload-bootstrap

Footer and forms need to be fixed too.

**Comment by Dreadchild** [comment]
Still not working this is sad...2 years after...

**Comment by lenamtl** [comment]
Hi I did a forked on latest Bootstrap v3 almost completed, stay tuned.

**Comment by ilackskills** [comment]
Status? Otherwise I will create a responsive theme and update my page.

**Comment by lenamtl** [comment]
My fork is on  hold, so create your own theme.

As Projectsend got rewrited  to use PDO I have to recompare files, I'm waiting a more stable version prior doing that.

There are a lot of changes when switching to Bootstrap 3 not only Bootstrap syntax, jQuery version need to be update and most of the jquery plugin need to be update or replaced. This is a lot of work.

**Comment by Dreadchild** [comment]
why are you not just calling the script url from jQuery server?

**Comment by lenamtl** [comment]
Because Several jquery script/widget used in Projectsend are old and are not compatible with the jquery version required by bootstrap. 
So we have to find replacement  script for each, which I have done and it's required a lot of configuration. Then we need to change all the syntax of the pages, because Bootstrap 2 is very different from Bootstrap 3 especially with forms.

So now I'm waiting for a more statble version of the latest version that use PDO.
To update it with all the work I did with previous version.
Also I was facing an issue when deleting files even with all patches and fixes and I pretty sure it is related to all the changes I did.

So unfortunately this is not something that's only take a link changes ....
